packages/playground/src/utils/delete_deployment.ts

```diff
@@ -39,6 +39,14 @@ export async function deleteDeployment(grid: GridClient, options: DeleteDeployme
 
   /* Delete deployment */
   if (options.k8s) {
+    const { gateways } = await loadDeploymentGateways(grid, { filter: gw => true });
+    for (const gateway of gateways) {
+      try {
+        await grid.gateway.delete_name({ name: gateway.name });
+      } catch (error) {
+        console.log("Error while deleting k8s gateway.", error);
```

Comment:
```suggestion
        console.error("Error while deleting k8s gateway.", error);
```
to be more clear on the logger